include/deal.II/fe/fe_nothing.h

```diff
@@ -82,10 +82,12 @@ class FE_Nothing : public FiniteElement<dim,spacedim>
 public:
 
   /**
-   * Constructor. Argument denotes the number of components to give this
-   * finite element (default = 1).
+   * Constructor. First argument denotes the number of components to give this
+   * finite element (default = 1). Second argument decides whether FE_Nothing
+   * will dominate any other FE in compare_for_face_domination() (default = false).
    */
-  FE_Nothing (const unsigned int n_components = 1);
+  FE_Nothing (const unsigned int n_components = 1,
+              const bool dominate = false);
```

Comment:
> yes, that's my intention.

Can you describe it in these terms in the documentation then?

> btw, note that cases like |FESystem(FE_Q(1),1,FE_Nothing,1)| vs
> |FESystem(FE_Q(2),1,FE_Q(1),1)| will already result in constraining Q1 to be
> come zero at the interface as the first FE will dominate.

I don't think so. Since we work on vector components individually, in the 
first vector components the Q1 dominates the Q2, and so at the interface 
between cells, the first vector component will be linear. For the second 
vector component, you have an FE_Nothing and a Q1 coming together, but neither 
dominates the other and so there are no continuity requirements at all -- the 
FE_Nothing side will of course be zero, but the Q1 side can be whatever it 
wants. The solution will in general be discontinuous there.